model/src/main/kotlin/licenses/ResolvedCopyright.kt

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Is it worth turning these into a separate package @sschuberth @mnonnenmacher ?

Reply:
Let's see... maybe some more refactoring will be coming up from my side, and maybe that shows whether it's worth it.